src/content/docs/dev/amplifier/its-hub/introduction.mdx

```diff
@@ -0,0 +1,37 @@
+# ITS Hub
+
+import { Callout } from "/src/components/callout";
+
+<Callout>
+  The Interchain Token Service (ITS) Hub is currently under active development,
+  please check back frequently for updates.
+</Callout>
+
+The Interchain Token Service (ITS) Hub is a Cosmwasm [contract](https://github.com/axelarnetwork/axelar-amplifier/tree/feat/its-hub/interchain-token-service) on the Axelar network that acts as a central routing hub for token transfers across Amplifier and consensus chains.
+The ITS Hub is designed to enhance the security, interoperability, and extensibility of cross-chain token transfers by ensuring that all ITS-enabled chains route their token transactions through this hub.
+This approach helps maintain consistent security measures, enables scalability, and supports a diverse set of connected chains, including [EVM](/resources/contract-addresses/mainnet/#evm-contract-addresses) and non-smart contract chains.
+
+Axelar supports two types of tokens: **Gateway Tokens** and **ITS Tokens**.
+
+- **Gateway Tokens**: Cross-chain tokens deployed natively via the Axelar protocol and the old EVM Axelar gateway.
+  These are legacy tokens that the [ITS Edge Contracts](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) support. They are native tokens registered and minted within the Axelar consensus model.
+- **ITS Tokens**: Tokens deployed via ITS and controlled by the ITS protocol. These tokens are based on [General Message Passing (GMP)](/dev/general-message-passing/overview/) and are minted, burned, and managed on the connected [EVM chains](/resources/contract-addresses/mainnet/#evm-contract-addresses) directly.
+
+The ITS Hub supports both token types and enables the secure bridging of these tokens across different types of chains.
```

Comment:
This isn't right, is it? I thought ITS Hub didn't support Gateway tokens, can you confirm?

Reply:
I was informed it's going to support but not the initial release; I am double-checking again with Milap. If it going to support then i might just need to add a caveat.